fedimint-wasm-tests/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "fedimint-wasm-tests"
 version = "0.0.0"
 edition = "2021"
 license = "MIT"
+description = "Wasm tests for the fedimint."
 publish = false
```

Comment:
This crate isn't being published, I don't think this is needed.

Reply:
> This crate isn't being published, I don't think this is needed.

@elsirion , It was my  first thought but all other unpublishable crates have `discription` field :

![image](https://github.com/fedimint/fedimint/assets/133089376/d847664c-4e1f-4701-b77c-4a07586ec481)

Reply:
I mean ok, now that it's already open, but in general this isn't worth the team's bandwidth it consumes in review.

Reply:
> I mean ok, now that it's already open, but in general this isn't worth the team's bandwidth it consumes in review.

MB, I understands -> will  try to avoid it .